mongodb-dev/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM        ubuntu:14.04
+MAINTAINER  Apiary <sre@apiary.io>
+
+ENV REFRESHED_AT 2015-02-23
+
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+RUN echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' | tee /etc/apt/sources.list.d/10gen.list
+
+RUN apt-get update && apt-get install -y mongodb-org=2.6.7 mongodb-org-server=2.6.7 mongodb-org-shell=2.6.7 mongodb-org-mongos=2.6.7 mongodb-org-tools=2.6.7
+
+RUN mkdir -p /data/db
+
+EXPOSE 27017
+
+ENTRYPOINT usr/bin/mongod --smallfiles
```

Comment:
Fixed in https://github.com/apiaryio/docker-base-images/commit/29616678a0e6b2bae51567aba4f60bc0606e3f9d